conjurapi/authn.go

```diff
@@ -48,30 +48,30 @@ func (c *Client) createAuthRequest(req *http.Request) error {
 	return nil
 }
 
-func (c *Client) Authenticate(loginPair authn.LoginPair) ([]byte, error) {
-	var (
-		req *http.Request
-		err error
-	)
-
-	if c.config.V4 {
-		req, err = wrapper_v4.AuthenticateRequest(c.config.ApplianceURL, loginPair)
-	} else {
-		req, err = wrapper.AuthenticateRequest(c.config.ApplianceURL, c.config.Account, loginPair)
+func (c *Client) Authenticate(loginPair authn.LoginPair) (io.ReadCloser, error) {
```

Comment:
One go convention that I think is a pretty good idea is to doc the functions/data structures you export from your libraries (uppercase names). Can we start to do that? I think it would make this code easier to read through.